lib/pure/net.nim

```diff
@@ -533,14 +545,13 @@ when defineSsl:
       if SSL_CTX_check_private_key(ctx) != 1:
         raiseSSLError("Verification of private key file failed.")
 
-  proc newContext*(protVersion = protSSLv23, verifyMode = CVerifyPeer,
+  proc newContext*(minProtVersion = protTLS, verifyMode = CVerifyPeer,
```

Comment:
Nit: I don't know how much we care about argument name breakage, @Araq? @narimiran?

Reply:
This is a proc that has many named arguments, so maybe out of the abundance of caution we could keep the name as-is? It would still make sense with that name.

Reply:
> Nit: I don't know how much we care about argument name breakage

According to a quick search on github, very few projects explicitly set protVersion

Reply:
> This is a proc that has many named arguments, so maybe out of the abundance of caution we could keep the name as-is? It would still make sense with that name.

I changed the name since this is a behavior change. Applications that previously set this parameter (which restrict the TLS version to an exact one), will want to check whether they still want the one version limit (in which case they will probably file an issue to have it back), or opt-in into the new behavior (which now enforce only the minimum).